README.md

```diff
@@ -466,6 +466,19 @@ detected by GIR is different than what was implemented and conversion to the C
 types are wrong in autogenerated functions that have such objects as argument.
 This can be overridden with the `ref_mode` configuration.
 
+Getters are automatically renamed to comply with Rust codying style guidelines.
+However, this can cause name clashes with existing functions. If you want to
+bypass the automatic renaming mechanism, use `bypass_auto_rename = true`:
+
+```toml
+[[object]]
+name = "Gtk.TextBuffer"
+[...]
+    [[object.function]]
+    name = "get_insert"
+    # Avoid clash with the `insert` operation.
+    bypass_auto_rename = true
```

Comment:
That's exactly what I'm afraid of in the end... Two questions about this:

1. How much name conflicts would be there?
2. Couldn't we automatically detect those conflicts?

Reply:
> How much name conflicts would be there?

See https://github.com/gtk-rs/gtk-rs/pull/211/commits/4fb1e7b4b61f3ada35cee9988a58b9a3b48639aa & https://github.com/gtk-rs/gtk-rs/pull/211/commits/021ee0930b9cd7dd4448fa3647a095c0334e70e2. Please take a look at https://github.com/gtk-rs/gtk-rs/pull/211.

> Couldn't we automatically detect those conflicts?

That's already been done for `get_property_...`, others need some kind of human cognition IMHO.